dojo/api_v2/views.py

```diff
@@ -25,6 +25,7 @@
 from dojo.api_v2 import serializers, permissions
 
 
+
```

Comment:
I can approve that! :-D

Reply:
Actually, I am surprised that flake8 is ok with 3 lines.

Reply:
I think I even added it because flake in my visual studio code complained about it.